<launch>
    <node name="april_tag_pointcloud" pkg="rr_evgp" type="april_tag_pointcloud" output="screen">
        <param name="camera_frame" type="str" value="camera_optical"/>
        <param name="pointcloud" type="str" value="/april_tag_pointcloud/pointcloud"/>
        <param name="destination_frame" type="str" value="base_footprint"/>
        <param name="tag_detections_topic" type="str" value="/tag_detections"/>
        <param name="tag_detection_markers" type="str" value="/tag_detection_markers"/>
        <param name="opponents_april" type="str" value="/opponents_april"/>
        <param name="x_offset" type="int" value="0"/>
        <param name="y_offset" type="int" value="0"/>
        <param name="px_per_m" type="int" value="10"/>
        <param name="width" type="double" value="1"/>
        <param name="height" type="double" value="1.5"/>
    </node>
</launch>
